modules/dnn/test/test_layers.cpp

```diff
@@ -777,6 +777,50 @@ TEST(Layer_PriorBox, squares)
     normAssert(out.reshape(1, 4), target);
 }
 
+TEST(Layer_Test_DWconv_Prelu, Accuracy)
+{
+    // Test case
+    // input       img size 3x16x16  value all 1
+    //   |
+    //   v
+    // dw_conv     weight[0]=-1 weight[1]=-2 weight[2]=-3 no_bias
+    //   |
+    //   v
+    // prelu       weight={1,2,3}
+    //   |
+    //   v
+    // output      out size 3x14x14  if right: out[0]=-9 out[0]=-36 out[0]=-81 
+    //             but current opencv output: out[0]=-27 out[0]=-54 out[0]=-81
+
+    Net net;
+    {
+        const string proto = findDataFile("dnn/conv_prelu.prototxt", false);
+        const string model = findDataFile("dnn/conv_prelu.caffemodel", false);
```

Comment:
Please build a model in runtime using `LayerParams` and `addLayerToPrev` methods. This way `.prototxt` and `.caffemodel` are not required.
https://github.com/opencv/opencv/blob/4ec456f0a0993457120b916b36bde1d6df93e37b/modules/dnn/test/test_layers.cpp#L581-L597

Reply:
done.

Reply:
@dkurt